packages/wdio-browserstack-service/src/Percy/Percy.ts

```diff
@@ -114,24 +123,43 @@ class Percy {
 
     async fetchPercyToken() {
         const projectName = this.#projectName
-
+        let data = {
+            enabled: this.percyAutoEnabled,
+            token: null,
+            percyCaptureMode: this.percyCaptureMode
+        }
         try {
             const type = this.#isApp ? 'app' : 'automate'
-            const response = await nodeRequest(
-                'GET',
-                `api/app_percy/get_project_token?name=${projectName}&type=${type}`,
+            let query = 'api/app_percy/get_project_token?'
+            if (projectName) {
+                query += `name=${projectName}&`
+            }
+            if (type) {
+                query += `type=${type}&`
+            }
+            if (this.#options.percy !== undefined) {
```

Comment:
let's send Percy undefined when `percy` not set

Reply:
As this is a GET API, we can't send "undefined" in query params, hence removing the percy value completely from query